src/components/organisms/PostsContainerOrganism.tsx

```diff
@@ -0,0 +1,29 @@
+import { api } from "~/utils/api";
+import SyncLoader from "react-spinners/SyncLoader";
+
+const PostsContainerOrganism = () => {
+  const {data, isLoading} = api.posts.getAll.useQuery();
+  
+  // Early return
+  if(!data) {
+    return <div>Loading...</div>
+  }
+
+  if (isLoading) {
+    return <SyncLoader size={5} color={"#FFCD00"} />
+  }
+
```

Comment:
```suggestion
const posts = data.map(({ post, author }) => (
  <div key={post.id}className="p-8 border-b gap-4 border-slate-700 flex items-center">
      <img 
        className="w-12 h-12 rounded-full"
        src={author?.imageUrl} 
        alt="author avatar" 
      />
      <div className="">{post.content}</div>
    </div>
))
```

前回のReviewの時に漏れてしまい申し訳ないです。
こちらのmapは変数に一旦しましょう！

なぜかというとn`PostContainerOrganism`を使用した時に、
型が写真のようになってしまうからです！
もし難しいようであればハドルで聞いてください💦

<img width="712" alt="スクリーンショット 2023-10-18 9 21 39" src="https://github.com/maria-skygrid/t3-intro/assets/100871461/41fa5ac6-bb9d-4cfc-b572-fe977fdc621d">